include/elements/element.h

```diff
@@ -48,6 +48,9 @@ class Element {
   //! Return number of shape functions
   virtual unsigned nfunctions() const = 0;
 
+  //! Return number of local shape functions
+  virtual unsigned nfunctions_local() const = 0;
```

Comment:
@bodhinandach This is just initialization, right?

Reply:
Yes, pure virtual functions.